src/apps/api/tests/test_competitions.py

```diff
@@ -69,3 +70,47 @@ def test_adding_organizer_accepts_them_if_they_were_existing_participant(self):
             competition=self.comp,
             status=CompetitionParticipant.APPROVED
         ).count() == 1
+
+
+class PhaseMigrationTests(APITestCase):
```

Comment:
also a quick test for only migrating parent submissions.

Reply:
added